rudder-agent/SOURCES/Makefile

```diff
@@ -193,9 +194,6 @@ endif
 	# Apply patches
 	for PATCHNAME in patches/cfengine/*.patch; do echo "Applying $$PATCHNAME..."; $(PATCH) -d ./cfengine-source -p1 < $$PATCHNAME; echo ""; done
 
-	# Bootstrap the package using autogen before compilation
-	cd cfengine-source && NO_CONFIGURE=1 ./autogen.sh
```

Comment:
Already done in the build-cfengine target (and there is no autogen.sh in the 3.7 tar)